libexec/pyenv-init

```diff
@@ -83,58 +83,51 @@ function help_() {
   ksh )
     profile='~/.profile'
     ;;
-  fish )
-    profile='~/.config/fish/config.fish'
-    ;;
   * )
     profile='your profile'
     ;;
   esac
 
-  { echo "# Add pyenv executable to PATH by adding"
-    echo "# the following to ~/.profile:"
-    echo
+  {
     case "$shell" in
     fish )
+      echo "# Add pyenv executable to PATH by running"
+      echo "# the following interactively:"
+      echo
       echo 'set -Ux PYENV_ROOT $HOME/.pyenv'
       echo 'set -Ux fish_user_paths $PYENV_ROOT/bin $fish_user_paths'
```

Comment:
These two need to only run once, even if you start a nested shell so that the `PATH` entries don't duplicate.